modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/MultiselectPicklistFDSPropsTransformer.tsx

```diff
@@ -3,6 +3,81 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import ClayButton from '@clayui/button';
```

Comment:
@Hi @Artartur, please rename this file and the component exported by default. This is not something exclusive for the Multiselect Picklist, so we should have a more generic name.